src/pages/dummy-voting.tsx

```diff
@@ -0,0 +1,29 @@
+import { header } from "@fonts";
+import { useState } from "react";
+import Bg from "~/components/background";
+import { api } from "~/utils/api";
+
+const createVote = api.voteTools.create.useMutation({})
```

Comment:
ini salah penempatan yaaa, coba dipindahin. dan sebenernya be ga perlu bikin react pagenya. tapi, kalau mau buat nyoba sih silakan aja